package.json

```diff
@@ -3,7 +3,7 @@
   "description": "React Native module for Radar, the leading geofencing and location tracking platform",
   "homepage": "https://radar.com",
   "license": "Apache-2.0",
-  "version": "3.5.6",
+  "version": "3.5.7",
```

Comment:
We need to run npm install so that this gets reflected in package-lock.json

Reply:
ok